generated/proto/persist_fs_commitlog.proto

```diff
@@ -0,0 +1,25 @@
+syntax = "proto3";
+package schema;
+
+message CommitLogInfo {
+	int64 start = 1;
+	int64 duration = 2;
+	int64 index = 3;
```

Comment:
It is in the filename, it's in here too so we check that filename matches the canonical source of info about the commit log which is in the proto contents.